scripts/coverage_report_jenkins.py

```diff
@@ -24,6 +48,19 @@ def group_list_dict_by(ld, by):
 
 def jenkins_artifact_url(jenkins_username, jenkins_token, jenkins_url, jenkins_job, jenkins_build,
         artifact_path):
+    """Build Jenkins artifact URL for a particular Jenkins job.
+
+    Args:
+        jenkins_username:  Jenkins login.
+        jenkins_token:  ???
```

Comment:
This is an API token, generated in the UI for a given user and used for client authentication. Is not used for UI login, and does not include user's password.

Reply:
:heavy_check_mark: